app/controllers/ChoiceController.scala

```diff
@@ -56,7 +56,7 @@ class ChoiceController @Inject()(
       case forms.Choice.AssociateUCR =>
         proceedJourney(AssociateUcrAnswers(), consolidations.routes.MucrOptionsController.displayPage())
       case forms.Choice.DisassociateUCR =>
-        proceedJourney(DisassociateUcrAnswers(), consolidations.routes.DisassociateDucrController.display())
+        proceedJourney(DisassociateUcrAnswers(), controllers.consolidations.routes.DisassociateUcrController.display())
```

Comment:
Can do `consolidations.routes.DisassociateUcrController.display()` (i.e. drop the `controllers`.  But this might be fixed when you update from master